docs/faq.md

```diff
@@ -80,3 +89,28 @@ Currently, the main benefit of [signed releases](/checks.md#signed-releases) is
 However, there are already moves to make it even more relevant. For example, the OpenSSF is working on [implementing signature verification for NPM packages](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/) which would allow a consumer to automatically verify if the package they are downloading was generated through a reliable builder and if it is correctly signed.
 
 Signing releases already has some relevance and it will soon offer even more security benefits for both consumers and maintainers.
+
+### Branch-Protection: How to setup a 10/10 branch protection on GitHub?
+
+To get a 10/10 score for Branch-Protection check using a non-admin token, you should have the following settings for your main branch:
```

Comment:
Doesn't Branch-Protection also run for release branches? (Release branches are detected only if the project uses GitHub Releases)

Reply:
I am not sure, so will change the text to "branches" to be more generic.